components/raftstore/src/store/snap.rs

```diff
@@ -1994,9 +1995,49 @@ impl TabletSnapManager {
         Ok(Self {
             base: path,
             receiving: Arc::default(),
+            stats: Arc::default(),
         })
     }
 
+    pub fn begin_snapshot(&self, key: TabletSnapKey, start: Instant, generate_duration_sec: u64) {
```

Comment:
Suggest to add an unit test for these new methods.

Reply:
already add unit test for it